src/proc.cr

```diff
@@ -62,7 +62,7 @@
 # ```
 # module Ticker
 #   # The callback for the user doesn't have a Void*
-#   @@box : Box(Int32 ->)
+#   @@box : Pointer(Void)?
```

Comment:
That's wrong, proper type is `Box`, see the [example](https://github.com/crystal-lang/crystal/blob/1e2f601f7c929916402c3adcccb1ad4d2329c94f/src/proc.cr#L70).